TBETA.ipynb

```diff
@@ -506,7 +511,66 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 42,
+   "execution_count": 302,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#A function identical to plot_numbers, but returns the dimensions of the LDOS and LCurrent plots.\n",
+    "#Additionally, it does not plot the bandstructure, and hides the numbered atom plot.\n",
+    "def plot_numbers_sys2():\n",
+    "        sites = list(sys2.sites()) #get the sites from kwant builder, the coordinates are shown in terms of lattice vectors\n",
+    "        fig, ax = plt.subplots() #only one subplot here\n",
+    "        ax.set_yticks([])\n",
```

Comment:
Are all lines here really needed? For example,  line 523 - ax.set_yticks is definitely not needed if we don't actually display the plot, same with plotting the "numbers" in lines ~535-549. Probably only need to get sites, make a subplot, and then do kwant.plotter in order to get xaxis2,yaxis2, no?